Sources/TuistLoader/Models+ManifestMappers/CoreDataModel+ManifestMapper.swift

```diff
@@ -12,7 +12,63 @@ extension TuistCore.CoreDataModel {
     static func from(manifest: ProjectDescription.CoreDataModel, generatorPaths: GeneratorPaths) throws -> TuistCore.CoreDataModel {
         let modelPath = try generatorPaths.resolve(path: manifest.path)
         let versions = FileHandler.shared.glob(modelPath, glob: "*.xcdatamodel")
-        let currentVersion = manifest.currentVersion
+
+        var currentVersion: String!
+        if let hardcodedVersion = manifest.currentVersion {
+            currentVersion = hardcodedVersion
+        } else {
+            currentVersion = try CoreDataVersionExtractor.version(fromVersionFileAtPath: modelPath)
+        }
         return CoreDataModel(path: modelPath, versions: versions, currentVersion: currentVersion)
     }
 }
+
+/// Extract version from .xccurrentversion file.
+private final class CoreDataVersionExtractor {
```

Comment:
This is a good candidate to be in `TuistSupport`. What about extracting it there into its own file `CoreDataVersionExtractor`?